Tests/IGListAdapterTests.m

```diff
@@ -41,7 +41,7 @@ - (void)setUp {
     self.adapter.collectionView = self.collectionView;
     self.adapter.dataSource = self.dataSource;
 
-    if (@available(iOS 11.0, *)) {
+    if (@available(iOS 11.0, tvOS 11.0, *)) {
```

Comment:
ummm, this is not ideal as this Test is highly optimized for testing for iOS. And the 2 test cases you moved are actually better coupled/co-located here within this file.

Plus, it adds a lot more maintenance cost in the future if people did not pay attention to some tests need to be put into the "XXX_IB" test files.